README.md

```diff
@@ -1,95 +1,99 @@
-Hardware
-========
+# Pixhawk Hardware Designs
 
-[![Join the chat at https://gitter.im/PX4/Hardware](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PX4/Hardware?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Join the chat at https://gitter.im/PX4/Hardware](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/PX4/Hardware?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge) 
 
-PX4 Hardware designs
+[Pixhawk](http://pixhawk.org) is an independent open-hardware project that aims to provide "the gold standard" for readily-available, hiqh-quality and low-cost autopilot hardware designs for the academic, hobby and developer communities.
```

Comment:
@LorenzMeier There are two main points here:
1. When you said "to provide the standard for" you could have meant "provide the definition" or "be the gold standard". I have assumed you mean the second one, and modified appropriately.
2. This mentions **for the academic, hobby and developer communities**. Do we care about industry, commerce, etc?